macros/schema_tests/cardinality_equality.sql

```diff
@@ -0,0 +1,44 @@
+{% macro test_cardinality_equality(model, from, to, field) %}
+
+with table_a as (
+select
+  {{ from }},
+  count(1) as num_rows
+from {{ model }}
+group by 1
+),
+
+table_b as (
+select
+  { field }},
+  count(1) as num_rows
+from {{ to }}
+group by 1
+),
+
+except_a as (
+  select *
+  from table_a
+  except
+  select *
+  from table_b
+),
+
+except_b as (
+  select *
+  from table_b
+  except
+  select *
+  from table_a
+)
+
+select count(1)
+from (
+  select *
+  from except_a
+  union all
+  select *
+  from except_b
+)
```

Comment:
If you alias this subquery, then this schema test will also work on Postgres. At present, it errors out with:
```
subquery in FROM must have an alias
  LINE 36: from (
                ^
  HINT:  For example, FROM (SELECT ...) [AS] foo.
```

So just change it to
```
) as sbq
```
and you should be good! Or, you could make another CTE for the subquery

Reply:
Ah, good catch. I just created an additional CTE for the subquery.